compiler+runtime/include/cpp/jank/detail/to_runtime_data.hpp

```diff
@@ -100,4 +102,15 @@ namespace jank::detail
   {
     return m;
   }
+
+  template <typename T>
+  object_ref to_runtime_data(native_vector<jtl::option<T>> const &m)
+  {
+    runtime::detail::native_persistent_vector ret;
+    for(auto const &e : m)
+    {
+      (void)ret.push_back(to_runtime_data(e));
+    }
+    return make_box<obj::persistent_vector>(ret);
+  }
```

Comment:
This can be removed by not using an option for catch bodies anyway.

Reply:
we don't have a "to_runtime_data" for native_vector, so we still need it.